azure-functions/src/bindings/durable_orchestration_client.rs

```diff
@@ -1,9 +1,41 @@
-use crate::{
-    durable::{CreationUrls, ManagementUrls},
-    rpc::{typed_data::Data, TypedData},
-};
+use crate::rpc::{typed_data::Data, TypedData};
+use azure_functions_durable::{OrchestrationClient, OrchestrationResult};
 use serde::Deserialize;
-use serde_json::from_str;
+use serde_json::{from_str, Value};
+
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct CreationUrls {
+    #[serde(rename = "createNewInstancePostUri")]
+    create_new_instance_url: String,
+    #[serde(rename = "createAndWaitOnNewInstancePostUri")]
+    create_new_instance_and_wait_url: String,
+}
+
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct ManagementUrls {
+    id: String,
+    #[serde(rename = "statusQueryGetUri")]
+    status_query_url: String,
+    #[serde(rename = "sendEventPostUri")]
+    raise_event_url: String,
+    #[serde(rename = "terminatePostUri")]
+    terminate_url: String,
+    #[serde(rename = "rewindPostUri")]
+    rewind_url: String,
+    #[serde(rename = "purgeHistoryDeleteUri")]
+    purge_history_url: String,
+}
+
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct BindingData {
+    #[serde(rename = "taskHubName")]
+    task_hub: String,
+    creation_urls: CreationUrls,
```

Comment:
I see inconsistency in providing a name for struct fields. Should we add it explicitly the same way as for other fields?

Reply:
Not sure I understand.  Are you referring to `task_hub`?

Reply:
`task_hub` already has `#[serde(rename = "taskHbuName")]`, but `creation_urls` and `management_urls` dosn't

Reply:
I'll make them consistent in the next PR.